app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsActionFragment.kt

```diff
@@ -118,4 +146,13 @@ class TransactionDetailsActionFragment : BaseViewBindingFragment<FragmentTransac
         item.value = value
         return item
     }
+
+    private fun getDivider(): View {
+        val item = View(requireContext())
+        val layoutParams = LayoutParams(MATCH_PARENT, dpToPx(1))
+        layoutParams.setMargins(0, dpToPx(16), 0, 0)
+        item.layoutParams = layoutParams
+        item.setBackgroundColor(ContextCompat.getColor(requireContext(), R.color.light_grey))
+        return item
```

Comment:
maybe using `View(requireContext()).apply { ... }` would be a bit more readable.

Reply:
maybe :)

Reply:
you'll have to cast layoutParams then in order to set margins. I would leave it as is. IMO it is straightforward and easy to read.